### PR TITLE
Fix cancel clearing and start investigating session cookie issue

### PR DESCRIPTION
**Pull request by coltborg**
* Remove cancelled file from array after it's been deleted
* Added a check to filter files from the session cookie that aren't in the db

### CODE REVIEW

File: src/main/java/formflow/library/FileController.java [hunk 1]
```diff
@@ -94,9 +94,10 @@ public ResponseEntity<?> upload(
     log.info("POST upload (url: {}): flow: {}, inputName: {}", request.getRequestURI().toLowerCase(), flow, inputName);
     try {
       if (!doesFlowExist(flow)) {
-        throwNotFoundError(flow, null, String.format("Could not find flow with name %s in your application's flow configuration.", flow));
+        throwNotFoundError(flow, null,
```
Comment (spokenbird) [inline src/main/java/formflow/library/FileController.java:97]: Is your styles/linter adding a line break?

File: src/main/java/formflow/library/FileController.java [hunk 2]
```diff
@@ -203,7 +205,8 @@ RedirectView delete(
       HttpServletRequest request
   ) {
     try {
-      log.info("POST delete (url: {}): fileId: {} inputName: {}", request.getRequestURI().toLowerCase(), fileId, dropZoneInstanceName);
+      log.info("POST delete (url: {}): fileId: {} inputName: {}", request.getRequestURI().toLowerCase(), fileId,
```
Comment (spokenbird) [inline src/main/java/formflow/library/FileController.java:208]: Similar question as above here?

File: src/main/resources/META-INF/resources/webjars/form-flow/0.0.1/js/qunit-upload-tests.js [hunk 3]
```diff
@@ -45,4 +45,4 @@ QUnit.module('Dropzone', function () {
 });
 QUnit.testDone(function () {
   document.querySelector("#uploadTest").className += " display-none";
-});
\ No newline at end of file
+});
```
Comment (spokenbird) [inline src/main/resources/META-INF/resources/webjars/form-flow/0.0.1/js/qunit-upload-tests.js:48]: Not actually sure what the difference is here?

File: src/main/java/formflow/library/FileController.java (hunk 1 above)
Comment (coltborg) [reply src/main/java/formflow/library/FileController.java:97]: Yep, this was the linter.

File: src/main/java/formflow/library/FileController.java (hunk 2 above)
Comment (coltborg) [reply src/main/java/formflow/library/FileController.java:208]: Yep, same, this was from the linter.

**Pull request merged**